### ISSUE TITLE
Sub-folders to Mods folder

### ISSUE THREAD

**Issue by SuperKirbylover** [coop-deluxe/sm64coopdx#227]
this is a really minor request, but i think personally would help me and friends a lot

with the amount of mods there are now, i think it would be really helpful to have sub-folders within the mods folder for different types of mods. for example, romhacks go in the `romhacks` folder, gamemodes go in the `gamemodes` folder. maybe even a `characterselect` folder, with how many character select mods there are now?
all other mods, will still go in the root of the mod folder. all mods can still be read there, but for organization purposes, having a system like that would be really helpful. i have way too many mods downloaded

**Comment by ThePlayerRolo** [comment]
yeah, this would be really helpful since there are already some special categories for romhacks.

**Comment by AgentXLP** [comment]
This is a good idea, although I'm unsure of how I would go about it, and I'm also worried about missing any edge cases. This would need to be a feature added in the future when the time is right.
